### ISSUE TITLE
Delayed ability to head control

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#10477]
**Preferred term label:**
Delayed ability to head control

**Definition (free text, please give PubMed ID)**
A failure to achieve the ability to maintain head in an upright midline position when held upright at an appropriate developmental stage. Most children can control their head to upright and midline position in about 3 to 4 months which continues to increase throughout the first year and beyond (PMID:22466026 and 33620859).

**Parent term (use hpo.jax.org/app)**
Delayed gross motor development (HP:0002194)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-6670-9157
